### PR TITLE
[Architecture] Define initial architecture

### PR DESCRIPTION
**Pull request by AndriyBosik**

### CODE REVIEW

**Review by ponomarovv — APPROVED** [review]
/

File: src/main/java/com/example/wiki/reactive/config/BeanConfig.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.example.wiki.reactive.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
```
Comment (AndriyBosik) [inline src/main/java/com/example/wiki/reactive/config/BeanConfig.java:8]: adsfasdfasdf

Comment (AndriyBosik) [inline src/main/java/com/example/wiki/reactive/config/BeanConfig.java:8]: asdfasdfa

File: src/main/java/com/example/wiki/reactive/controller/ExampleController.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import com.example.wiki.reactive.model.RecentChange;
+import com.example.wiki.reactive.service.ApiService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Flux;
+
+@RestController
```
Comment (AndriyBosik) [inline src/main/java/com/example/wiki/reactive/controller/ExampleController.java:12]: asdfasdfa

File: src/main/java/com/example/wiki/reactive/model/RecentChange.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
```
Comment (AndriyBosik) [inline src/main/java/com/example/wiki/reactive/model/RecentChange.java:11]: asdfasdfa

**Pull request merged**